Block/Sales/View.php

```diff
@@ -55,49 +35,58 @@ public function _construct()
      */
     public function getCheckoutOptionsHtml()
```

Comment:
De return type voor deze functie is niet (meer) correct nu, graag aanpassen (`void` als het goed is) -- zie ook opmerking hieronder.